### DISCUSSION TITLE
HTTPS or FTP for PPO In Network Rate File?

### DISCUSSION THREAD

**Discussion by OHCDiana** [CMSgov/price-transparency-guide#384] [Q&A] [answered]
We are a PPO Network for a single state. Very often, we are wrapped by a larger national network. Sometimes we wrap a smaller network. Since our In Network Rate file will not be the final and complete file eventually published by the plan sponsor or issuer, is it acceptable for us to use FTP rather than HTTPS to send our portion of the network to the payor for aggregation? Any other insights regarding PPO compliance would be appreciated.

**Comment by shaselton-usds** [comment, marked as answer]
@OHCDiana All files are required to be delivered via HTTPS as noted in the [documentation](https://github.com/CMSgov/price-transparency-guide#transport-mechanism---https).
